### ISSUE TITLE
mxnet Parser has not supported operator upsampling

### ISSUE THREAD

**Issue by liupengkd** [microsoft/MMdnn#741]
when I want to convert mnxet model(including upsampling with nearest) to tensorflow but I get this error.
mxnet Parser has not supported operator upsampling

**Comment by rikichou** [comment]
I have the same problem

**Comment by liupengkd** [comment]
@rikichou I changed the source code of mxnet parser ignoring these operators not supported including upsampling, crop, reshape and SoftmaxActivation. Then I get .npy and .py and modify .py to add these layer by hand。

**Comment by rikichou** [comment]
@liupengkd great !  and can you show me your .py ?

**Comment by StanislasBertrand** [comment]
Hi @liupengkd, could you give an example of a modification you made for an operation in the mxnet parser ? I'm having some trouble doing the change myself.


By the way, the SotfmaxAxctivation in the symbol file can be changed to softmax (new mxnet operation name) for it to work.
